roles/ceph-common/defaults/main.yml

```diff
@@ -215,10 +201,14 @@ rbd_client_directory_mode: null
 
 rbd_client_log_path: /var/log/ceph
 rbd_client_log_file: "{{ rbd_client_log_path }}/qemu-guest-$pid.log" # must be writable by QEMU and allowed by SELinux or AppArmor
+<<<<<<< HEAD
```

Comment:
I think we want to keep ``rbd_client_admin_socket_path``, but lose the rest of the ``rbd_*`` options here.